openedx/stanford/djangoapps/auth_lagunita/models.py

```diff
@@ -3,10 +3,13 @@
 Record extra lagunita-centric user data
 """
 from __future__ import unicode_literals
+import logging
 
 from django.conf import settings
 from django.db import models
+from django.db.models import Q
 
+log = logging.getLogger('auth_lagunita')
```

Comment:
This doesn't appear to be used anywhere in this file?

Reply:
I must have factored out the logging calls in this file;
I removed this.